src/rewriter.coffee

```diff
@@ -197,7 +202,7 @@ exports.Rewriter = class Rewriter
         #  1. We have seen a `CONTROL` argument on the line.
         #  2. The last token before the indent is part of the list below
         #
-        if prevTag not in ['=>', '->', '[', '(', ',', '{', 'TRY', 'ELSE', '=']
+        if prevTag not in ['=>', '->', '[', '(', ',', '{', 'ELSE', '=']
```

Comment:
safe to remove `TRY` from this list b/c an implicit call could never be at the top of the stack (ie `inImplicitCall()` could never be true) when we've just seen `TRY` b/c the preceding clause would've caused a `CONTROL` to get pushed onto the stack (in the previous `scanTokens()` iteration). Effectively, this list can safely be mutually exclusive from the list above (`['IF', 'TRY', ...]`)